actions/mysql_explain_action.go

```diff
@@ -55,7 +55,7 @@ func NewMySQLExplainAction(id string, params *agentpb.StartActionRequest_MySQLEx
 	if params.TlsFiles != nil && params.TlsFiles.Files != nil {
 		err := tlshelpers.RegisterMySQLCerts(params.TlsFiles.Files)
 		if err != nil {
-			log.Error(err)
+			logrus.Error(err)
```

Comment:
Not sure if the move to logrus brings any UX change on binary output front. Let me know if this switch is okay.

Reply:
Yeah I don't think it does either. This was the only place we used it and it wasn't customized in any way. Prometheus' common/log used to be a wrapper around logrus and has been deprecated for a while, it just got removed recently (https://github.com/prometheus/common/pull/306)